project/Dependencies.scala

```diff
@@ -110,7 +110,7 @@ object Dependencies {
     ))
 
   val CassandraVersionInDocs = "4.0"
-  val CassandraDriverVersion = "4.13.0"
+  val CassandraDriverVersion = "4.15.0"
   val CassandraDriverVersionInDocs = "4.13"
```

Comment:
should CassandraDriverVersionInDocs be updated too?

Reply:
yep, that might have gotten lost when I updated the PR. Will fix it.